src/routes/pages/expand.ts

```diff
@@ -28,6 +35,12 @@ route.get('/:group/:code', async (req, res) => {
       throw new Error('Shortcode not found')
     }
     res.redirect(url.longUrl)
+    // @ts-ignore
```

Comment:
why is ts-ignore needed here

Reply:
tsc is throwing error when I call URLs.increment

Reply:
That is because `increment` method exists on the instance of a model and not on the model class itself. Do not use `@ts-ignore` so liberally, it defeats the purpose of using Typescript. If you are required to fight Typescript, it means you are doing something wrong (this is precisely the advantage of using strong types)

Reply:
Even on the instance it was giving the same error and I found the cause. In the models.ts file
```
db.define<URLAttributes, URLAttributes>
```
this should be replaced with
```
db.define<Sequelize.Instance<URLAttributes> & URLAttributes, URLAttributes>
```
As the first argument is the interface of the instance and we are providing just the URLAttributes and hence it misses out on methods like increment() and save()

Reply:
Ok correct, https://michalzalecki.com/using-sequelize-with-typescript/